Docs/Design/Personas.md

```
@@ -0,0 +1,6 @@
+# Personas
+
+
+
+https://hub.vroid.com/en
+https://github.com/uwu/vrh-deobfuscator?tab=readme-ov-file
```

markdownlint-cli2 (0.21.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)